pkgs/games/deliantra/default.nix

```diff
@@ -0,0 +1,10 @@
+pkgs:
+
+let
+  callPackage = pkgs.callPackage;
+in {
+  deliantra-server = callPackage ./deliantra-server.nix {};
+  deliantra-arch = callPackage ./deliantra-arch.nix {};
+  deliantra-maps = callPackage ./deliantra-maps.nix {};
+  deliantra-data = callPackage ./deliantra-data.nix {};
+}
```

Comment:
All this could also be in top-level.

Reply:
It could be, but top-level is already so massive that I'd rather push the boilerplate down here.